pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -89,6 +89,28 @@ type ContainerReplicaStatus struct {
 	Image                string                  `json:"image"`
 	ImageID              string                  `json:"imageID"`
 	Started              *bool                   `json:"started,omitempty"`
+
+	Region string `json:"region,omitempty"`
+}
+
+// ForRegion for a ContainerReplica is called when the ContainerReplica is synced.
+// This ensures the proxy can find the correct ContainerReplica for exec/logs.
+func (in *ContainerReplica) ForRegion(region string) bool {
```

Comment:
NIT: The naming "ForX" isn't intuitive to me. Would make sense to be "IsRegion" and "IsNotRegion". Unsure if there is some naming convention or use for using "For", so might be missing something.

Reply:
The reason I used `ForRegion` is that certain objects (like cluster compute classes) actually modify the object to be for the region, other just do a check that would be more like and "is region" approach. I thought `ForRegion` was a good middle ground.

Reply:
Could we reframe the method as `SetRegion(region string) bool` to make it clear at the call site that a mutation _could_ take place?